src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx

```diff
@@ -139,6 +139,12 @@ Dictionary
             <legend>Concepts (HEAD version)</legend>
             <p className="paragraph">
               <b>Total Concepts</b>
+:
+              {' '}
+              {active_concepts}
```

Comment:
Is `active_concepts` the same as total concepts?
I've noticed that it is repeated below. Shouldn't we just keep one if they are the same?

Reply:
@brucemakallan no it's not the same. There's another [PR](https://github.com/openmrs/openmrs-ocl-client/pull/308) that fixes that issue.

Reply:
OK. Thanks